saleor/giftcard/notifications.py

```diff
@@ -11,8 +18,16 @@ def send_gift_card_notification(user, app, email, gift_card, manager):
             "app_id": app.id if app else None,
             "app_name": app.name if app else None,
         },
+        "recipient": {
+            "name": _get_recipient_name(email),
```

Comment:
There is a bunch of logic in `user.get_full_name()` that joins first and last name, fallbacks to name from user's billing address or an email). I'm wondering if we should use it or rather just return the first name and last name as it is, and let the template author decide how to handle rendering? @patrys What do you think?

https://github.com/mirumee/saleor/blob/377481193f203d28678396a8ae44a5581ce5c1bc/saleor/account/models.py#L243